e2e/motoko/Main.mo

```diff
@@ -1,23 +1,23 @@
+import EvmRpcProductionFiduciary "canister:evm_rpc";
 import EvmRpcStaging13Node "canister:evm_rpc_staging_13_node";
 import EvmRpcStagingFidicuary "canister:evm_rpc_staging_fiduciary";
-import EvmRpcProductionFiduciary "canister:evm_rpc";
 
 import Blob "mo:base/Blob";
+import Buffer "mo:base/Buffer";
 import Debug "mo:base/Debug";
 import Cycles "mo:base/ExperimentalCycles";
 import Principal "mo:base/Principal";
 import Text "mo:base/Text";
-import Buffer "mo:base/Buffer";
 import Evm "mo:evm";
 
 shared ({ caller = installer }) actor class Main() {
 
     type TestCategory = { #staging; #production };
 
-    type SubnetTarget = (Nat, Nat);
+    type SubnetTarget = (Text, Nat32, Nat);
     // (`nodes in subnet`, `expected cycles for JSON-RPC call`)
```

Comment:
The comment should be moved up one line and extended.

Reply:
Thanks; updated.